src/extension/background-script/db.ts

```diff
@@ -14,7 +14,7 @@ class DB extends Dexie {
       allowances:
         "++id,&host,name,imageURL,tag,enabled,totalBudget,remainingBudget,lastPaymentAt,lnurlAuth,createdAt",
       payments:
-        "++id,allowanceId,host,location,name,description,totalAmount,totalFees,preimage,paymentRequest,paymentHash,destination,createdAt",
+        "++id,allowanceId,host,location,name,description,totalAmount,totalFees,preimage,paymentRequest,metadata,paymentHash,destination,createdAt",
     });
     this.version(2).stores({
```

Comment:
the DB above must never be changed. 
in Dexie you always create new versions to update. 
check the docs: https://dexie.org/docs/Dexie/Dexie.version()#sample

Reply:
making this change